.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report or request for help
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
```

Comment:
Since the category is issues, and not always bugs, what if we had:
**Describe the issue**
A clear and concise description of what the issue is.